lib/sparrow/fcm/v1/notification.ex

```diff
@@ -115,20 +115,34 @@ defmodule Sparrow.FCM.V1.Notification do
     %{notification | project_id: project_id}
   end
 
-  @spec verify(t) :: t | {:error, :invalid_notification}
-  def verify(notification) do
-    data = Enum.map(notification.data, &verify_value/1)
+  @spec normalize(t) :: t | {:error, :invalid_notification}
+  def normalize(notification) do
+    with {:ok, notification} <- do_normalize(notification),
+         {:ok, android} <- Sparrow.FCM.V1.Android.normalize(notification.android),
+         {:ok, webpush} <- Sparrow.FCM.V1.Webpush.normalize(notification.webpush) do
+      {:ok,
+       List.foldl([{:android, android}, {:webpush, webpush}], notification, fn {k, v}, acc ->
+         Map.put(acc, k, v)
+       end)}
+    else
+      _ -> {:error, :invalid_notification}
```

Comment:
Isn't that the only possible value? If so, the whole `else` clause is superfluous.